### PR TITLE
Unify PerformanceTest metric dashboard naming and series

### PR DESCRIPTION
**Pull request by Abacn**
Part of #18204

* Tests are named after 'IO Connector | Specifications | data size'.

* Python performance tests plot read and write time in same plot as in Java

**Please** add a meaningful description for your change here

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by Abacn** [comment]
R: @pabloem 

Is there a way to show the preview? Previewed locally but without actual data because had no access to the database.

**Comment by Abacn** [comment]
Have setup a dumb influxdb database and verified the rendering:
image

**Comment by Abacn** [comment]
Java IO dashboard also checked. Ready for review now.

**Comment by pabloem** [comment]
I am not familiar with this beyohnd the normal docs here: https://cwiki.apache.org/confluence/display/BEAM/Community+Metrics

**Comment by pabloem** [comment]
are you sure we won't need the configuration sections that you removed? are these autopopulated?

**Comment by Abacn** [comment]
> 

Yeah, thanks. I followed this documentation and get it working except I am not using the real data from influxDB. The screenshot above shows the results of some manipulated data.

The removed sections are combining  read and write time in same plot as in Java. Currently they are plotted separately: http://104.154.241.245/d/gP7vMPqZz/python-io-it-tests-dataflow?orgId=1

I am working on adding more Python performance tests and this is a clean up for current frontend as a preclude.

File: .test-infra/metrics/grafana/dashboards/perftests_metrics/Java_IO_IT_Tests_Dataflow.json [hunk 1]
```diff
@@ -855,7 +855,7 @@
       "yaxes": [
         {
           "$$hashKey": "object:403",
-          "format": "s",
+          "format": "none",
```
Comment (Abacn) [inline .test-infra/metrics/grafana/dashboards/perftests_metrics/Java_IO_IT_Tests_Dataflow.json:858]: This figure plots count instead of time. Fixed here.

**Comment by Abacn** [comment]
Thanks @pabloem . Replied above. PTAL

**Comment by Abacn** [comment]
SparkReceiverIO Performance test has been merged. Rebasing and also changing the naming of it.

**Comment by Abacn** [comment]
R: @aromanenko-dev

**Comment by pabloem** [comment]
thanks @Abacn - this LGTM : )

**Pull request merged**